### ISSUE TITLE
Allow Custom Header Color in embeds

### ISSUE THREAD

**Issue by awong-dev** [findthemasks/findthemasks#1050]
Request for OSMS to allow embeds to specify banner color via query params.

Two considerations:  (a) if the embedder can change it to any color, how will it interact with the fonts  (b) will this cause any confusion to the user about where the data is coming from or who it might be going to?  There is actually a small security/privacy benefit to having a strongly distinct branding...

![find-the-masks-embed-requests](https://user-images.githubusercontent.com/144579/84589445-5dbbed80-ade3-11ea-97dd-d01fb684d8c1.jpg)

**Comment by awong-dev** [comment]
One thought is maybe we shoudl watermark FTM on the masks proper?  Goal again is to ensure people know wehre teh data is going/coming to/from.

**Comment by linderr** [comment]
OSMS has cut the embed height and added their own footer so the redirect is not entirely where all the data lives. https://opensourcemedicalsupplies.org/local-response/

**Comment by tobiasdeml** [comment]
Yeah we did this as a temporary solution to not confuse the local response groups since they've been asked for local response data in this form from the get-go; the goal would be to have something from FTM that can have an OSMS logo slapped next to it so the user knows the data goes to both orgs at the same time. Basically a shared intake form that is auto-customized to the partner org depending on who embeds it (via a URL parameter in the embed).
I figured this is a more complicated ask, so I left it out of our first set of requests, so that was the reason for the embed-overlay solution (that's temporary until we can figure out the dual intake experience).

**Comment by tobiasdeml** [comment]
Basically we imagine that the footer link would go to this page, but instead of our airtable intake form, it'd be the FTM intake with a OSMS customization that explains to the user that their info will be shared between the two orgs.
https://opensourcemedicalsupplies.org/local-response/register/
